doc/content/reference/configuration/identity-server.md

```diff
@@ -3,3 +3,105 @@ title: "Identity Server Options"
 description: ""
 weight: 2
 ---
+
+## Database Options
+
+The Identity Server needs to be connected to a PostgreSQL-compatible database. Details for the form of the URI can be found in the [PostgreSQL documentation](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING).
+
+- `is.database-uri`: Database connection URI
+
+## Email Options
+
+The Identity Server can be configured with different providers for sending emails. Currently the `sendgrid` and `smtp` providers are implemented.
+
+- `is.email.provider`: Email provider to use
+
+When `sendgrid` is used as provider, an API key is required. For testing, use the sandbox to prevent emails from actually being sent.
+
+- `is.email.sendgrid.api-key`: The SendGrid API key to use
+- `is.email.sendgrid.sandbox`: Use SendGrid sandbox mode for testing
+
+When `smtp` is used as provider, provide the address of the SMTP server (`host:port`), as well as the username and password for the SMTP server.
+
+- `is.email.smtp.address`: SMTP server address
+- `is.email.smtp.username`: Username to authenticate with
+- `is.email.smtp.password`: Password to authenticate with
+- `is.email.smtp.connections`: Maximum number of connections to the SMTP server
+
+The email address and name of the sender should be configured regardless of the provider that is used.
+
+- `is.email.sender-address`: The address of the sender
+- `is.email.sender-name`: The name of the sender
+
+Most emails contain the name of the network and links to the Identity Server or Console.
+
+- `is.email.network.name`: The name of the network
+- `is.email.network.identity-server-url`: The URL of the Identity Server
+- `is.email.network.console-url`: The URL of the Console
+
+Although The Things Stack comes with a number of builtin email templates, it is possible to override those with custom templates. You can specify either a directory or a URL to load templates from. For mor information on email templates, see the [email templates reference]({{< relref "../email-templates" >}}).
```

Comment:
`built-in` (Compound adjective before a noun) 👨‍🏫

Reply:
Let's try to move fast today. We can do a big spelling/grammar round some time.